settings.js

```diff
@@ -117,6 +117,15 @@ function generate(){
             "trim"
           ]
         },
+        "peliasUnit": {
+          "type": "custom",
+          "tokenizer":"keyword",
+          "char_filter" : ["alphanumeric"],
```

Comment:
@trescube do you think we would ever need non-alphanumeric characters in unit numbers? Maybe `-` or `/`?

Reply:
We've had a discussion and are pretty sure analyzing alphanumeric characters only will work well enough to start :)